### PR TITLE
Issue/478 update index.rst

### PR DESCRIPTION
**Pull request by FloLey**
# Description

update the index.rst file with then changes of the file in inmanta-service-orchestrator

closes https://github.com/inmanta/inmanta-service-orchestrator/issues/478

# Self Check:

Strike through any lines that are not applicable (`~~line~~`) then check the box

- [x] Attached issue to pull request
- [x] Changelog entry
- [ ] Type annotations are present
- [ ] Code is clear and sufficiently documented
- [ ] No (preventable) type errors (check using make mypy or make mypy-diff)
- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)
- [ ] Correct, in line with design
- [ ] End user documentation is included or an issue is created for end-user documentation (add ref to issue here: )
- [ ] If this PR fixes a race condition in the test suite, also push the fix to the relevant stable branche(s) (see [test-fixes](https://internal.inmanta.com/development/core/tasks/build-master.html#test-fixes) for more info)

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -48,41 +48,50 @@
 
     * **End-to-end**: Inmanta Service Orchestrator ensures end-to-end consistency, higher flexibility and a shorter time to cash by enabling end-to-end automation of all service delivery aspects:
 
-    * Multi-domain: designed to interact across physical and virtual domains, such as WAN, edge, access network, NFV, cloud, containers, and datacenter.
-    * Holistic: A single, unifying automation solution, providing service orchestration, network orchestration, NFV orchestration (NFVO), as well as generic VNF management (gVNFM), cloud orchestration and configuration management. No other automation tools required.
-    * Full lifecyle: Manage advanced service lifecycle, covering creation, on-boarding, provisioning, modification, scaling, upgrading and decommissioning.
+      * Multi-domain: designed to interact across physical and virtual domains, such as WAN, edge, access network, NFV, cloud, containers, and datacenter.
```
Comment (sanderr) [inline docs/index.rst:51]: The indentation fixes you added here are definitely good.

File: docs/index.rst [hunk 2]
```diff
@@ -48,41 +48,50 @@
...
 
 .. toctree::
-    :maxdepth: 1
+    :maxdepth: 3
 
     quickstart
     install
     architecture
     language
+    moduleguides
```
Comment (sanderr) [inline docs/index.rst:76]: I don't think the pages `moduleguides` and `extensions` even exist. If you build from this branch, does it look any different than the build from master?

File: docs/index.rst [hunk 3]
```diff
@@ -48,41 +48,50 @@
...
+      * Interoperability through pluggable adapters and open APIs
+      * API-ification of orchestrated services to easily plug services into the OSS/BSS environment
+      * Support for brownfield environments by fine-grained roll-out
 
     The Inmanta Service Orchestrator product is based on mature technology backed by 15+ years of research and interaction with companies offering telecom and cloud services.
 
 
 .. toctree::
-    :maxdepth: 1
+    :maxdepth: 3
```
Comment (sanderr) [inline docs/index.rst:70]: The ToC looks very polluted to me with a depth of 3. I personally don't see any value in it. I'd ask Bart when he's back to be sure (I wouldn't block the merge on it though, just make sure to ask if he has an opinion when he's back). @arnaudsjs do you have an opinion on this?

File: docs/index.rst [hunk 4]
```diff
@@ -48,41 +48,50 @@
...
     troubleshooting
     changelogs
 
+Additional resources
+--------------------
+
+* `Inmanta User Mailinglist <https://groups.google.com/forum/#!forum/inmanta-users>`_
+* `Inmanta Developer Mailinglist <https://groups.google.com/forum/#!forum/inmanta-devel>`_
+* `Inmanta Twitter <https://twitter.com/inmanta_com>`_
+
```
Comment (sanderr) [inline docs/index.rst:94]: This section I really don't know about. I think it's good to add it back, but again I'd ask Bart when he's back.

File: changelogs/unreleased/478-update-index-rst.yml [hunk 5]
```diff
@@ -0,0 +1,5 @@
+description: Update index.rst with so that it contains the changes that where do to the same file in inmanta-service-orchestrator
+issue-nr: 478
+issue-repo: inmanta-service-orchestrator
+change-type: patch
+destination-branches: [master, iso7]
```
Comment (sanderr) [inline changelogs/unreleased/478-update-index-rst.yml:5]:
```suggestion
destination-branches: [master, iso7, iso6]
```

While iso6 does already use this file, I think it should also get these fixes.

File: docs/index.rst (hunk 2 above)
Comment (FloLey) [reply docs/index.rst:76]: indeed I don't think they exist, I removed them

File: docs/index.rst (hunk 3 above)
Comment (FloLey) [reply docs/index.rst:70]: I prefer it with 1 to.

Comment (arnaudsjs) [reply docs/index.rst:70]: I agree that 1 is better than 3.

File: docs/index.rst (hunk 4 above)
Comment (FloLey) [reply docs/index.rst:94]: @bartv do you have an opinion on this?

Comment (bartv) [reply docs/index.rst:94]: These mailing lists are not in use anymore so I would just remove the entire section.

**Review by sanderr — APPROVED** [review]

**Pull request closed without merging**